### PR TITLE
sync pdf and build scripts to GNU ARM Embedded Toolchain 2016q2-20160622

### PR DESCRIPTION
**Pull request by soarqin**

### CODE REVIEW

**Comment by yifanlu** [comment]
The commits seem to indicate that there's more changes than just that. Please either include the full set of changes in your description or open a separate pull request for each set of changes.

**Comment by soarqin** [comment]
there are only 3 files changed, I did a full diff and stripped 2 changes for gdb build, as vitasdk does not build gdb and removed commands from script. All other changes are applied.
If you need the full diff to preview, I can put it later as soon as I getting back to my computer.

**Comment by yifanlu** [comment]
Oh I see, this is for gcc5.4. Okay let me test it.

**Comment by soarqin** [comment]
[2015q2-2016q2-diff.txt](https://github.com/vitasdk/buildscripts/files/403156/2015q2-2016q2-diff.txt)
full diff attached

File: build-toolchain.sh [hunk 1]
```diff
@@ -496,7 +498,11 @@ if [ "x$DEBUG_BUILD_OPTIONS" = "x" ] ; then
       *)        PERM="/111" ;;
     esac
 
-    STRIP_BINARIES=`find $INSTALLDIR_NATIVE/lib/gcc/$TARGET/$GCC_VER/ -maxdepth 1 -name \* -perm $PERM -and ! -type d`
+    if [ "x$BUILD" == "xx86_64-apple-darwin10" ]; then
+        STRIP_BINARIES=`find $INSTALLDIR_NATIVE/lib/gcc/$TARGET/$GCC_VER/ -maxdepth 1 -name \* -perm +111 -and ! -type d`
+    else
+        STRIP_BINARIES=`find $INSTALLDIR_NATIVE/lib/gcc/$TARGET/$GCC_VER/ -maxdepth 1 -name \* -perm /111 -and ! -type d`
+    fi
```
Comment (endrift) [inline build-toolchain.sh]: Why is this here? It was already fixed by previous PRs. (Also this would only work with one specific--and old--version of OS X.)

**Comment by soarqin** [comment]
I just try to keep build scrips in-sync of newest ARM Toolchain so that it would be easy to apply changes in future official toolchains

**Comment by soarqin** [comment]
okay, here it is. I removed darwin changes from patch, keeps other changes from 2016q2

**Comment by soarqin** [comment]
closed due to #23
